src/components/CookieBanner/CookieBanner.js

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from 'react';
+import { CgClose } from "react-icons/cg";
+import CookieIcon from '../../img/cookieIcon.png'
+import VhiLogo from '../../img/cookieVhiLogo.png'
+
+export default function CookieBanner() {
+
+    const [isVisible, setIsVisible] = useState(false);
```

Comment:
the indentation on this file looks like 4 spaces but the CookieBanner.test.js seems to have 2 spaces

Reply:
Fixed this now.